test/context_test.exs

```diff
@@ -2,6 +2,8 @@ defmodule Sentry.ContextTest do
   use ExUnit.Case
   import Sentry.TestEnvironmentHelper
 
+  doctest Sentry.Context, except: [{:add_breadcrumb, 1}]
```

Comment:
`Sentry.Context.add_breadcrumb/1` adds timestamps automatically, and so the doctest for it will basically never pass :(.